services/API-service/src/api/admin-area-data/dto/upload-admin-area-data.dto.ts

```diff
@@ -26,3 +37,27 @@ export class UploadAdminAreaDataDto {
   @IsNumber()
   public value: number;
 }
+
+export class UploadAdminAreaDataJsonDto {
+  @ApiProperty({ example: 'ETH' })
+  @IsNotEmpty()
+  @IsString()
+  public countryCodeISO3: string;
+
+  @ApiProperty({ example: 3 })
+  @IsNotEmpty()
+  @IsNumber()
+  public adminLevel: number;
```

Comment:
use [`AdminLevel`](https://github.com/rodekruis/IBF-system/blob/3b31eb37140e6046f0938d2063f036aa623ee57b/services/API-service/src/api/country/admin-level.enum.ts) instead of number.